services/scan/src/store.ts

```diff
@@ -432,6 +446,7 @@ export class Store {
         frontFinishedAdjudicationAt ?? null,
         backFinishedAdjudicationAt ?? null
       );
+      this.invalidateBatchesExportedAt();
```

Comment:
That is how it worked before d2fe24b0223dc1277930a73dc0a919dbffaf06bb, but as I was working on invalidating the export when _deleting a batch_, I realized the approach of marking individual batches with `exported_at=null` would not work in that case, because each remaining batch would still have the `exported_at` column set.

As an alternative approach, we could use a single flag `exported_at` instead of `exported_at` on each batch. That is effectively how `batches.exported_at` works in this current state, anyway.

I guess before going any further, I should also confirm that deleting a batch should invalidate the export. What do you think @eventualbuddha? If not, I can revert d2fe24b0223dc1277930a73dc0a919dbffaf06bb and go back to individually invalidating batches.